css/home.css

```diff
@@ -182,6 +191,14 @@
     max-width: 350px;
     height: 198px;
   }
+
+  .homepage-hero-scroll__lookdown{
```

Comment:
please change this name to `.homepage-hero__anchor`

Reply:
done